src/libostree/ostree-repo-pull.c

```diff
@@ -2995,6 +2998,9 @@ _ostree_repo_remote_new_fetcher (OstreeRepo *self, const char *remote_name, gboo
       }
   }
 
+  _ostree_fetcher_set_low_speed_limit (fetcher, (low_speed_limit) ? low_speed_limit : OPT_LOWSPEEDLIMIT_DEFAULT);
```

Comment:
`(low_speed_limit) ? *low_speed_limit : OPT_LOWSPEEDLIMIT_DEFAULT` will probably help

Reply:
I removed all the pointers from the 3 options I added. Hopefully it looks better now?